osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/security/providers/JreEncryptionProvider.java

```diff
@@ -45,21 +46,25 @@ public JreEncryptionProvider(File keyStoreFile) {
         }
     }
 
-    protected Cipher getCipher() {
+    @Override
+    protected Cipher getCipher() throws EncrypterException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.opensmartgridplatform.shared.exceptionhandling.EncrypterException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)